apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -729,9 +738,21 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
                             <Message className="display-flex" size="small" warning>
                                 <Icon name="warning sign" color="orange" corner />
                                 <Message.Content className="tiny">
-                                    { t(translateKey + "modals.scan.regenerateWarning") }
+                                    { t(commonConfig.accountSecurityPage.mfa.totp.regenerateWarning) }
                                 </Message.Content>
                             </Message>
+                            {
+                                commonConfig.accountSecurityPage.mfa.totp.showRegenerateConfirmation
+                                    ? (
+                                        <Checkbox
+                                            className="mb-2"
+                                            label={ "Confirm regenerating a new QR code" }
```

Comment:
Can we add i18n strings here?

Reply:
Fixed with ad6160723f62e367fb76ac2aaed99ceaa4e9e986